sentry-okhttp/src/main/java/io/sentry/okhttp/SentryOkHttpEventListener.kt

```diff
@@ -406,6 +406,7 @@ public open class SentryOkHttpEventListener(
 
     private fun canCreateEventSpan(): Boolean {
         // If the wrapped EventListener is ours, we shouldn't create spans, as the originalEventListener already did it
-        return originalEventListener !is SentryOkHttpEventListener
+        return originalEventListener !is SentryOkHttpEventListener &&
```

Comment:
```suggestion
		// In case SentryOkHttpEventListener from sentry-android-okhttp is used, the is check won't work so we check for the class name as well.
        return originalEventListener !is SentryOkHttpEventListener &&
```

Reply:
Things that come to mind:
- Does this work when using ProGuard / R8?
- Can there be intermediate listeners that are sitting between this one and our other one?

Reply:
intermidiate listeners are fine
Regarding Proguard, let me check it 😅

Reply:
I added a proguard rule to keep the name of the old SentryOkHttpEventListener